uiautomation/src/main.rs

```diff
@@ -19,10 +24,10 @@ fn main() -> Result<()> {
         println!("window name: {}", name);
 
         // Query for WinRT API (will fail on earlier versions of Windows)
-        let element: AutomationElement = element.cast()?;
+        let element2: AutomationElement = element.cast()?;
```

Comment:
Why did this get renamed?

Reply:
If we really want to improve the sample, we should conditionally use the result of the cast and print a diagnostic if it fails to indicate that the API is not available.